components/grid/toolbar.md

```diff
@@ -10,208 +10,150 @@ position: 45
-        }
-
-        public static async Task<SampleData> Update(SampleData itemToUpdate)
-        {
-            var index = _data.FindIndex(i => i.ID == itemToUpdate.ID);
-            if (index != -1)
-            {
-                _data[index] = itemToUpdate;
-                return await Task.FromResult(_data[index]);
-            }
-
-            throw new Exception("no item to update");
+            itemForEdit.AgeInYears = argsItem.AgeInYears;
+            itemForEdit.HireDate = argsItem.HireDate;
+            itemForEdit.Name = argsItem.Name;
         }
     }
-}
-````
-
->caption The result from the code snippet above, after the built-in Create button in the toolbar was clicked
-
-![Blazor Create Toolbar Button](images/create-toolbar-button.png)
 
-## Custom Commands
-
-You can use the toolbar to add buttons that invoke actions specific to your application.
-
->caption How to define a custom command in the grid toolbar
+    public class Person
+    {
+        public int? EmployeeId { get; set; }
 
-````RAZOR
-@result
+        public string Name { get; set; }
 
-<TelerikGrid Data=@MyData Pageable="true" PageSize="15">
-	<GridToolBarTemplate>
-		<GridCommandButton Command="MyToolbarCommand" OnClick="@MyCommandFromToolbar" Icon="@SvgIcon.InfoCircle">Fire My Command</GridCommandButton>
-	</GridToolBarTemplate>
-	<GridColumns>
-		<GridColumn Field=@nameof(SampleData.Name) Title="Employee Name" />
-		<GridColumn Field=@nameof(SampleData.HireDate) Title="Hire Date" />
-	</GridColumns>
-</TelerikGrid>
+        public int? AgeInYears { get; set; }
 
-@code {
-	string result;
-
-	private void MyCommandFromToolbar(GridCommandEventArgs args)
-	{
-		//note - the args.Item object is null because the command item is not associated with an item
-
-		result = "my custom toolbar command fired at " + DateTime.Now.ToString();
-
-		StateHasChanged();
-	}
-
-	//in a real case, keep the models in dedicated locations, this is just an easy to copy and see example
-	public class SampleData
-	{
-		public int ID { get; set; }
-		public string Name { get; set; }
-		public DateTime HireDate { get; set; }
-	}
-
-	public IEnumerable<SampleData> MyData = Enumerable.Range(1, 50).Select(x => new SampleData
-	{
-		ID = x,
-		Name = "name " + x,
-		HireDate = DateTime.Now.AddDays(-x)
-	});
+        public DateTime HireDate { get; set; }
+    }
 }
 ````
 
->caption The result from the code snippet above, after the custom command button in the toolbar was clicked
-
-![Blazor Custom Command Toolbar](images/custom-command-toolbar.png)
+## Custom Toolbar Configuration
 
-## Custom Layout
+Add a `<GridToolBarTemplate>` tag inside `<TelerikGrid>` to configure a custom toolbar. You can add your own HTML and components to create a more complex layout in the grid header to match your business needs and also `GridCommandButton` instances (read more about the features available in those buttons in the [Command Column](slug://components/grid/columns/command) article).
```

Comment:
Also describe the difference in keyboard navigation behavior.

Use the component names in uppercase - Grid.